plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -101,70 +100,60 @@ public void testPartitionPushDown()
     public void testOffsetPushDown()
     {
         createMessages(topicNameOffset);
-        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _partition_offset between 2 and 10", topicNameOffset), 18);
-        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _partition_offset > 2 and _partition_offset < 10", topicNameOffset), 14);
-        assertProcessedInputPossitions(format("SELECT count(*) FROM default.%s WHERE _partition_offset = 3", topicNameOffset), 2);
-    }
-
-    private void assertProcessedInputPossitions(String sql, long expectedProcessedInputPositions)
-    {
-        DistributedQueryRunner queryRunner = getDistributedQueryRunner();
-        assertEventually(() -> {
-            MaterializedResultWithQueryId queryResult = queryRunner.executeWithQueryId(getSession(), sql);
-            assertEquals(getQueryInfo(queryRunner, queryResult).getQueryStats().getProcessedInputPositions(), expectedProcessedInputPositions);
-        });
+        Session session = getSession();
```

Comment:
We don't need to pass the default session by adding another `assertProcessedInputPositions`. See other comment.

Reply:
OK